ksql-serde/src/main/java/io/confluent/ksql/serde/json/KsqlJsonDeserializer.java

```diff
@@ -132,12 +132,18 @@ private Object enforceFieldType(Schema fieldSchema, JsonNode fieldJsonNode) {
       Iterator<String> fieldNames = jsonNode.fieldNames();
       while (fieldNames.hasNext()) {
         String fieldName = fieldNames.next();
-        keyMap.put(fieldName.toUpperCase(), fieldName);
+        if (fieldName.startsWith("@")) {
```

Comment:
Think i said this on another PR - is it possible that the field name is `@` in which case the next line would fail

Reply:
Yes, this is from the other PR. We will fail if the field name is `@` anyway.